digitalpublicgoods/kobotoolbox.json

```diff
@@ -0,0 +1,92 @@
+    ]
+  },
+  "NonPII": {
+    "collectsNonPII": "Yes",
+    "checkNonPIIAccessMechanism": "Yes",
+    "nonPIIAccessMechanism": "Users are able to access and export their data in a variety of formats, including JSON, CSV, XLS, geoJSON."
+  },
+  "privacy": {
+    "isPrivacyCompliant": "Yes",
+    "privacyComplianceList": [
+      "GDPR"
+    ],
+    "adherenceSteps": [
+      "- Consultations with specialists and lawyers",
+      "- Terms of service: https://www.kobotoolbox.org/terms/",
+      "- Privacy policy: https://www.kobotoolbox.org/privacy/"
+    ]
+  },
+  "standards": {
+    "supportStandards": "Yes",
+    "standardsList": [
+      "Support for W3 accessibility standards is being applied progressively"
+    ],
+    "evidenceStandardSupport": [
+      ""
+    ],
+    "implementBestPractices": "Yes",
+    "bestPracticesList": [
+      "User interface design principles"
+    ]
+  },
+  "doNoHarm": {
+    "preventHarm": {
+      "stepsToPreventHarm": "Yes",
+      "additionalInfoMechanismProcessesPolicies": "We try to educate users about the proper way to collect data, to protect their accounts, and we take steps to disable accounts when informed of any issues such as account passwords having been shared. \n Privacy ploicy: https://www.kobotoolbox.org/privacy"
+    },
+    "dataPrivacySecurity": {
+      "collectsPII": "Yes",
+      "typesOfDataCollected": [
+        "User's email addresses are mandatory when creating accounts. Users may voluntarily provide other details. Users further use KoBoToolbox to collect any data that they require for their own purposes, which may include PII; only users have access to this data. "
+      ],
+      "thirdPartyDataSharing": "No",
+      "dataSharingCircumstances": [
+        ""
+      ],
+      "ensurePrivacySecurity": "Yes",
+      "privacySecurityDescription": "Regular monitoring of dependencies, secure access to server resources, education of users about encryption-at-rest options, etc. More details available if needed. \nPrivacy policy links: \n- https://www.humanitarianresponse.info/en/applications/kobotoolbox/privacy-policy \n- https://www.kobotoolbox.org/privacy"
+    },
+    "inappropriateIllegalContent": {
+      "collectStoreDistribute": "Yes",
+      "type": "Users may collect text data, audio files, images, or video files as required by their projects. None are available for public distribution, so the following questions do not apply.",
+      "contentFilter": "",
+      "policyGuidelinesDocumentationLink": "",
+      "illegalContentDetection": "No",
+      "illegalContentDetectionMechanism": ""
+    },
+    "protectionFromHarassment": {
+      "userInteraction": "No",
+      "addressSafetySecurityUnderageUsers": "",
+      "stepsAddressRiskPreventSafetyUnderageUsers": [],
+      "griefAbuseHarassmentProtection": "",
+      "harassmentProtectionSteps": [
+        ""
+      ]
+    }
+  },
+  "locations": {
+    "developmentCountries": [
+      ""
+    ],
+    "deploymentCountries": [
```

Comment:
Though not required, it would be really nice to have countries listed here. Thoughts? @nathanbaleeta

Reply:
Resolved in commit f65b831